src/redux/actions/signupAction.js

```diff
@@ -0,0 +1,31 @@
+import { toast } from 'react-toastify';
+import * as actions from './actionTypes';
+
+export const succesfulSignUpAction = (data) => ({
+  type: actions.SIGNUP_SUCCESS,
+  payload: data,
+});
+
+export const failedSignUpAction = (error) => ({
+  type: actions.SIGNUP_ERROR,
+  payload: error,
+});
+
+// eslint-disable-next-line import/prefer-default-export
+export const signupAction = (input) => (dispatch) => fetch('https://codepirates-ah-backend-staging.herokuapp.com/api/v1/users/signup', {
```

Comment:
I think it would be better if you add base URL in the actiontypes file as BASE_URL as suggested by the TTL.

Reply:
fixed